cypress/integration/chi-js/chi-drawer.spec.js

```diff
@@ -0,0 +1,127 @@
+describe('chi-drawer', function() {
+  before(() => {
+    cy.visit('/tests/js/chi-drawer.html');
+  });
+
+  describe('Test open & close behavior of Left positioned Drawer', () => {
+    it('Click on Drawer trigger opens the Drawer', () => {
+      cy.get('#drawer-trigger-1').click();
+      cy.get('#drawer-1')
+        .should('have.class', '-active')
+        .should('have.class', '-left')
+        .should('be.visible');
+    });
+    it('Click on X button closes the Drawer and check for other functionalities', () => {
+      cy.get('#drawer-1')
+        .find('.-close')
+        .click()
+        .parent('.chi-drawer')
+        .should('not.be.visible')
+        .get('#drawer-1')
```

Comment:
Is this line necessary? I think the current selector is the same element, right?